lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -71,14 +80,28 @@ class HelpPanel extends React.Component {
           css={`
             direction: ltr;
             min-width: 295px;
-            padding: 90px 20px;
+            padding: 84px 25px;
             padding-bottom: 0px;
             display: inline-block;
           `}
         >
-          <Text heading>We&apos;re here to help</Text>
-          <Spacing bottom={5} />
-          <Text medium>
+          <Text
+            css={`
+              color: #404855;
+              font-weight: 800;
+            `}
+            heading
+          >
+            We&apos;re here to help
+          </Text>
+          <Spacing bottom={4} />
+          <Text
+            medium
+            css={`
+              color: #404855;
+              line-height: 18px;
```

Comment:
The default is
```
font-size: 13px;
line-height: 20px;
```

Now that Vic's asked to bump it up to 14px, medium + 18 becomes
```
font-size: 14px;
line-height: 18px;
```

Let's remove this and let it flow back to 20px (in the design the RHS stuff is 20px too)
Can you check w/ @vicborgy on this? The